packages/generator-wcfactory/generators/factory/templates/package.json

```diff
@@ -50,7 +50,8 @@
     "@storybook/addons": "4.0.7",
     "@storybook/cli": "4.0.7",
     "@storybook/polymer": "4.0.7",
-    "@emotion/core": "10.0.0-beta.8",
+    "@emotion/core": "0.x.x",
```

Comment:
Is the intent to force the version to be less than 1.0.0?
Should this be anything in version 10? `^10.0.0`?

Reply:
The intend was to use the exact same version definition has the warning that is shown whenever versions are checked for @emotion/core.  In practice, this still fetched version 0.13.1 so having a rule definition as ```^0.10``` would be equivalent.